test.js

```diff
@@ -1 +1,10 @@
-console.log('Hello World')
+// the hello world program
+console.log('Hello World');
```

Comment:
Codacy has a fix for the issue: [Replace `'Hello·World'` with `"Hello·World"`](https://app.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions-2/pullRequest?prid=30888)
```suggestion
console.log("Hello World");
```